addons/mail/models/mail_channel.py

```diff
@@ -445,9 +522,27 @@ def _notify_email_recipient_values(self, recipient_ids):
     def _notify_thread(self, message, msg_vals=False, **kwargs):
         # When posting a message on a mail channel, manage moderation and postpone notify users
         if not msg_vals or msg_vals.get('moderation_status') != 'pending_moderation':
-            super(Channel, self)._notify_thread(message, msg_vals=msg_vals, **kwargs)
+            # link message to channel
+            rdata = super(Channel, self)._notify_thread(message, msg_vals=msg_vals, **kwargs)
+
+            message_format_values = message.message_format()[0]
+            bus_notifications = self._channel_message_notifications(message, message_format_values)
+            self.env['bus.bus'].sudo().sendmany(bus_notifications)
+
+            # Message from mailing channel should not make a notification in Odoo for users
+            # with notification "Handled by Email", but web client should receive the message.
+            # To do so, message is still sent from longpolling, but channel is marked as read
+            # in order to remove notification.
+            for channel in self.filtered(lambda c: c.email_send):
+                users = channel.channel_partner_ids.mapped('user_ids')
+                for user in users.filtered(lambda u: u.notification_type == 'email'):
+                    channel.with_user(user).channel_seen(message.id)
```

Comment:
There is a performance concern here: https://github.com/odoo/odoo/pull/36000#issuecomment-668077789
Not sure if should be part of current task or not

Reply:
Well that's not really part of this PR. I suppose this can be done in future work on channels (like renaming models, ...).